torchtune/config/_utils.py

```diff
@@ -17,7 +17,7 @@
 
 def log_config(recipe_name: str, cfg: DictConfig) -> None:
     """
-    Logs the parsed config to rank zero.
+    Logs the resolved config to rank zero.
```

Comment:
Can you say what resolved means here? e.g. A resolved config is the provided config updated with any command line overrides.